### ISSUE TITLE
Comparison with shanshan sync?

### ISSUE THREAD

**Issue by chmac** [zokugun/vscode-sync-settings#4]
I found your extension looking for an option to sync my settings via git. The shanshan plugin uses a GitHub gist, which is prone to being overwritten, and there's no easy way to do merges, etc. It looks like your plugin will be a great fit for my needs, so firstly, thank you kindly for writing and publishing it.

Would you be open to adding a section to the readme which compares this plugin with others? The shanshan plugin does have >3m downloads, so in my mind it makes sense to explain to people what the differences are.

**Comment by daiyam** [comment]
For remote git, the extension will:
- pull the remote repository into a directory in the vscode's global storage
- sync the settings/files from vscode to that directory
- push the changes to the remote repository

The extension needs only the url and the branch to pull and push. No authentifications are stored. But the `git` command on your system will need to be able to read/write on the remote repository.


Ya, a comparison would be nice since I did tried several extensions before writing mine...

**Comment by chmac** [comment]
Nice. I assume I could also use the file sync option and then just track that directory with git.

Do you want me to leave this issue open to add a comparison to the readme?

**Comment by daiyam** [comment]
> Nice. I assume I could also use the file sync option and then just track that directory with git.

Yes, with https://github.com/zokugun/vscode-sync-settings#file or https://github.com/zokugun/vscode-sync-settings#local-git depending on your use case.

**Comment by TonyGravagno** [comment]
Since this ticket is open, I'll add that comparison with the VSCode built-in feature "Settings Sync" might be helpful.
https://code.visualstudio.com/docs/editor/settings-sync

**Comment by chmac** [comment]
There's nothing about this issue that is stale I'd say, but perhaps the author would like to close it as won't fix?

**Comment by daiyam** [comment]
Ya, I won't make any comparison in the README.
